test/helper/launcher/launcher.go

```diff
@@ -0,0 +1,257 @@
+package launcher
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"strings"
+)
+
+const (
+	ExpectedContext = "kind-kind"
+
+	LauncherTestInstall = "test-ako"
+
+	HelmRepoURL = "https://mongodb.github.io/helm-charts"
+
+	RepoRef = "mongodb"
+
+	OperatorChart = "mongodb-atlas-operator"
+
+	AtlasURI = "https://cloud-qa.mongodb.com"
+
```

Comment:
nit: let's remove the superfluous newlines between consts.

Reply:
coming next